pages/api/play-attempt/index.ts

```diff
@@ -8,6 +8,53 @@ import getTs from '../../../helpers/getTs';
 
 const MINUTE = 60;
 
+export async function forceUpdateLatestPlayAttempt(userId:string, levelId:string, context:AttemptContext, ts:number) {
```

Comment:
will this ever be called with an attemptContext other than just beaten?

Reply:
Probably not to be honest. I guess i could refactor this.